OpenRA.Mods.Common/Widgets/Logic/Ingame/GameInfoLogic.cs

```diff
@@ -29,7 +29,7 @@ public GameInfoLogic(Widget widget, World world, IngameInfoPanel activePanel)
 			widget.IsVisible = () => activePanel != IngameInfoPanel.AutoSelect;
 
 			// Objectives/Stats tab
-			var scriptContext = world.WorldActor.TraitOrDefault<LuaScript>();
+			var scriptContext = world.WorldActor.TraitsImplementing<LuaScript>().FirstEnabledTraitOrDefault();
```

Comment:
This would be better written as
```csharp
var hasError = world.WorldActor.TraitsImplementing<LuaScript>()
	.Any(sc => !sc.IsTraitDisabled && sc.FatalErrorOccurred);
```